### PR TITLE
[vcpkg docs] Translate README to Portuguese - Brazil and I've made step by step how install in Fedora

### PR DESCRIPTION
**Pull request by FeliPoxZ**
### I've made the translation of English documentation to Brazil Portuguese.
I guess the vcpkg is extreme important in labour market because it's make easier work with libraries in C and C++. A lot of C e C++ devs in my country use this tool but most of them don't known english so, to make easier the understading of how start and how works the tool for brazilians devs I made the translation.

 look: Some docs what the documentations linked still not translated.

### I've made a step by step how install vcpkg in Fedora Linux and how install the developer tool (equivalent in ubuntu) in Fedora.
 Those files are already in English and Brazilian Portuguese

### CODE REVIEW

**Review by vicroms — DISMISSED** [review]
I can understand enough to know that the content here is the same as in the English README but I cannot vouch for the grammar and spelling.

File: README_pt_BR.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+* Discord: [\#include \<C++\>](https://www.includecpp.org), no canal #🌏vcpkg
+* Documentos: [Documentation](docs/README.md)
+
+# Sumário
+
+- [Vcpkg: Visão Geral](#vcpkg)
+- [Sumário](#sumário)
+- [Primeiros Passos](#primeiros-passos)
+  - [Inicio rapido: Windows](#guia-de-inicio-rapido-windows)
+  - [Inicio rapido: Unix](#guia-rapido-de-inicio-unix)
```
Comment (vicroms) [inline README_pt_BR.md:25]:
```suggestion
  - [Inicio rapido: Windows](#guia-de-inicio-rapido-windows)
  - [Inicio rapido: Unix](#guia-de-inicio-rapido-unix)
```

Maybe we should make the headers consistent (needs to update the header below)

File: README_pt_BR.md [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+Para mias informações confira [Codigo de conduta][contributing:coc-faq]
+ou email [opencode@microsoft.com](mailto:opencode@microsoft.com)
+com quaisquer perguntas ou comentários adicionais.
+
+[contributing:submit-issue]: https://github.com/microsoft/vcpkg/issues/new/choose
+[contributing:submit-pr]: https://github.com/microsoft/vcpkg/pulls
+[contributing:coc]: https://opensource.microsoft.com/codeofconduct/
+[contributing:coc-faq]: https://opensource.microsoft.com/codeofconduct/
+
+# licença
```
Comment (valeriaconde) [inline README_pt_BR.md:341]: Licença

**Comment by MonicaLiu0311** [comment]
If you agree with the CLA, please reply with the following information.

`@microsoft-github-policy-service agree`

**Comment by MonicaLiu0311** [comment]
Pinging@FeliPoxZ 
Can you address the review suggestions? Thanks!

**Comment by dg0yt** [comment]
Maybe the new Fedora part should be left out of this PR.

**Comment by MonicaLiu0311** [comment]
Have you signed a CLA yet?

**Comment by MonicaLiu0311** [comment]
Is it still working (it hasn't progressed for a month)?

**Comment by MonicaLiu0311** [comment]
@FeliPoxZ 
Convert this PR to draft since there is no progress. Please ping us if this PR is ready for review again.

**Comment by vicroms** [comment]
The new section should be its own PR, we don't want README versions to start diverging too much. Otherwise it LGTM, except for the non-signed CLA.

**Comment by MonicaLiu0311** [comment]
Closing this PR since it seems that no progress is being made. Please ping us to reopen if work is still being done.

**Pull request closed without merging**